pkg/cfn/builder/vpc.go

```diff
@@ -128,7 +131,8 @@ func (c *ClusterResourceSet) addNATGateways() error {
 	case api.ClusterDisableNAT:
 		c.noNAT()
 	default:
-		return errors.New(*c.spec.VPC.NAT.Gateway + "is not a valid NAT gateway mode")
+		// TODO validate this before starting to add resources
+		return errors.New(*c.spec.VPC.NAT.Gateway + " is not a valid NAT gateway mode")
```

Comment:
Would you mind changing this to `fmt.Errorf`? It's currently preferred for new errors, we used `errors` package only for wrapping.

Reply:
Sure. BTW, this is from the original PR, I just added a whitespace.